src/interventions/android-webview-54.js

```diff
@@ -42,6 +42,6 @@ module.exports = (window, navigator) => {
 
   const regexResult = regexCheck.exec(userAgent);
   if (regexResult !== null) {
-    delete window.PaymentRequest;
+    window.PaymentRequest = null;
```

Comment:
The `delete` keyword leads to negative performance effects with V8's hidden classes and it's generally recommended to just set to `null` instead.  I would encourage setting the object to `null` here, in particular as you're working with the `window` object which will likely be regularly used during the page's lifecycle.